components/Shared/WYSYWYGComponent.tsx

```diff
@@ -0,0 +1,10 @@
+interface WYSYWYGProps {
+  content: string;
+  style: {
+    readonly [key: string]: string;
+  };
+}
+
+export function WYSIWYG({ content, style }: WYSYWYGProps) {
+  return <div dangerouslySetInnerHTML={{ __html: content }} />;
+}
```

Comment:
The styles are not being used. Put the `style.richtext` as a class on the div.
At a later time, we might switch to using a read-only Tiptap editor to display the WYSIWYG.